### PR TITLE
Logistics Officer Buff!!!! (Fuck you, leftover Yellowstone change)

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request

Tl;dr - Took the explosive granter off the construction loadout of CE (why did they have it, they already had the one with C4 with that trait.......... leftover Yellowstone change) and gave it instead to the Logistics Officer since they realistically should have it as they do gun crafting.

Also - it makes it so LO's can make mortar rounds now. Also-also, took away the X4 off CE since it's strong. Gave them 2 C4 instead.

## Why It's Good For The Game

LO buff omg

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog

:cl:
removes: Removed explosive crafting and BP off of CE construction loadout.
removes: Removed X4 off of CE explosive loadout option, replaced it with C4
add: Added explosive perk book to LO so they can craft mortars, grenades, and be useful. (I will buff them more soon so NCR has to use them more.)
/:cl:

### CODE REVIEW

**Review by maaacha — APPROVED** [review]

**Comment by Stutternov** [comment]
> so, the LO already has the explosives granter in their bag. it's just that their backpack is too full for it to spawn. if you clear out stuff they're granted via traits, such as gnb (which is now fixed), it should spawn

Amended

**Review by maaacha — APPROVED** [review]
